yocto/runqemu-smoke-test.sh

```diff
@@ -36,14 +36,8 @@ source oe-init-build-env ../../build
 # A positive exit code from now on is fatal
 set -e
 
-# Make sure we have the required utilities
-bitbake qemu-helper-native
-
 # Set up networking for qemu
-sudo ../sources/poky/scripts/runqemu-gen-tapdevs 1000 1000 4 tmp/sysroots-components/x86_64/qemu-helper-native/usr/bin
-
-# Give qemu permissions to access tun interfaces
-sudo setcap cap_net_admin+ep $(find tmp/work/ -name qemu-system-x86_64 | grep qemu-helper-native)
+sudo ../sources/poky/scripts/runqemu-gen-tapdevs `id -u` `id -g` 4 tmp/sysroots-components/x86_64/qemu-helper-native/usr/bin
```

Comment:
From buildlog:
`16:09:24  + sudo /workspace/pelux_yocto/sources/poky/scripts/runqemu-gen-tapdevs 1003 1003 4 tmp/sysroots-components/x86_64/qemu-helper-native/usr/bin
16:09:24  Error: tmp/sysroots-components/x86_64/qemu-helper-native/usr/bin/tunctl is not an executable
16:09:24  Usage: sudo /workspace/pelux_yocto/sources/poky/scripts/runqemu-gen-tapdevs <uid> <gid> <num> <staging_bindir_native>
16:09:24  Where <uid> is the numeric user id the tap devices will be owned by
16:09:24  Where <gid> is the numeric group id the tap devices will be owned by
16:09:24  <num> is the number of tap devices to create (0 to remove all)
16:09:24  <native-sysroot-basedir> is the path to the build system's native sysroot
16:09:24  For example:
16:09:24  $ bitbake qemu-helper-native
16:09:24  $ sudo /workspace/pelux_yocto/sources/poky/scripts/runqemu-gen-tapdevs 1003 1003 4 tmp/sysroots-components/x86_64/qemu-helper-native/usr/bin`

Reply:
Thanks, I have restored the line that builds qemu-helper-native now.